packages/react/src/themes/teams-dark/components/Button/buttonVariables.ts

```diff
@@ -6,20 +6,34 @@ export default (siteVars: any): Partial<ButtonVariables> => {
     colorActive: siteVars.colors.white,
     colorHover: siteVars.colors.white,
     colorFocus: siteVars.colors.white,
+    colorDisabled: siteVars.colors.grey[450],
     backgroundColor: 'transparent',
-    borderColorFocus: siteVars.black,
+    backgroundColorHover: siteVars.colors.grey[700],
```

Comment:
These keys seem a bit random to me. Are `450, 550, 850, etc` keys defined for all colors? How does the user know the color is defined or not without just trying and seeing what happens?

Reply:
The color palette guide will show the users all colors available per theme. No other idea at this moment (maybe we should have different names?) I am open for suggestions